### PR TITLE
Remove 2.5 support

### PR DESCRIPTION
**Pull request by zero-24**
This PR removes the 2.5 support for the plugin as 2.5 is EOS and it makes the plugin much smaller.

### how to test

make sure the plugin works with 3.3.6 and 3.4.0(RC)

### CODE REVIEW

**Comment by zero-24** [comment]
For reference see: https://github.com/joomla-extensions/install-from-web-client/commit/232318e69f6abb684a6dd76b86e75c04f3c995a6#diff-cc215dbf1ed448beb9f785b4a2f56531

File: plugins/installer/webinstaller/webinstaller.php [hunk 1]
```diff
@@ -251,10 +205,6 @@ private function getChanges()
 
 		if ($ishathor)
 		{
-			if (!$this->isJ25()) {
```
Comment (mbabker) [inline plugins/installer/webinstaller/webinstaller.php]: This if needs to be present (it was for 3.x support)

**Comment by beat** [comment]
Actually it's only this commit https://github.com/joomla-extensions/install-from-web-client/commit/232318e69f6abb684a6dd76b86e75c04f3c995a6 that needs to be reverted for achieving the removal of 2.5 support.

**Comment by beat** [comment]
But let's not change all at once. Prefering to concentrate on the very urgent server-side update to JED3.

File: plugins/installer/webinstaller/webinstaller.php (hunk 1 above)
Comment (zero-24) [inline plugins/installer/webinstaller/webinstaller.php]: re add the code with: https://github.com/zero-24/install-from-web-client/commit/aa764e7a707a479c89ca249ba20de243dec0ed76

**Comment by zero-24** [comment]
>Actually it's only this commit 232318e that needs to be reverted for achieving the removal of 2.5 support.

No the Commit also adds some RTL stuff :smile: We can of course close this PR and simple revert the commit. Feel free to do it. :smile:

**Comment by beat** [comment]
Okay, time to commit that one, so that other PRs can be adapted if needed.

Thanks @zero-24 for the work on this and @mbabker for the check.

**Pull request merged**